test/integration/project-page.test.js

```diff
@@ -198,8 +198,8 @@ describe('www-integration project-creation signed in', () => {
         // SauceLabs doesn't have access to the sb3 used in 'load project from file' test
         // https://support.saucelabs.com/hc/en-us/articles/115003685593-Uploading-Files-to-a-Sauce-Labs-Virtual-Machine-during-a-Test
         if (remote) {
-            await driver.get('https://github.com/LLK/scratch-www/blob/develop/test/fixtures/project1.sb3');
-            await clickText('Download');
+            await driver.get('https://github.com/scratchfoundation/scratch-www/blob/develop/test/fixtures/project1.sb3');
+            await clickXpath('//Button[@data-testid="download-raw-button"]');
```

Comment:
The button is (now?) an icon and does not contain the word "Download" as text. It has a distinctive `data-testid` attribute, though, so that's what I switched to. If they change the page in the future this could break, but hopefully before that happens we can find a better overall solution that doesn't involve going to GitHub at all.